source/chapters/configuration.rst

```diff
@@ -204,10 +210,98 @@ designed for demanding low latency audio programs like Mixxx. It can be
 difficult to setup JACK and PulseAudio to work well together. So, unless you
 already use JACK, it is easiest to let Mixxx suspend PulseAudio and use ALSA.
 
-.. warning:: On GNU/Linux it is not recommended to use the ``pulse`` device
-             with the ALSA Audio API. This is an emulation layer for ALSA
-             provided by PulseAudio. It has poor performance, but may be
-             suitable for basic playback.
+If the PulseAudio plugin for alsalibs is installed on GNU/Linux, you can 
+choose the virtual device ``pulse``. This allows Mixxx to share the default 
+system sound card with other media players. This does only work if you start 
+mixxx without pasuspender. Since the sound stream is routed from ALSA to Pulse 
```

Comment:
without pasuspender, which you can do by running "mixxx" from a console rather than clicking the launcher icon in a menu or on your desktop.